typing/jkind.ml

```diff
@@ -396,21 +396,11 @@ let raise ~loc err = raise (Error.User_error (loc, err))
 let relevant_axes_of_modality ~relevant_for_nullability ~modality =
   Axis_set.create ~f:(fun ~axis:(Pack axis) ->
       match axis with
-      | Modal axis -> (
+      | Modal axis ->
         let (P axis) = Mode.Const.Axis.alloc_as_value (P axis) in
         let modality = Mode.Modality.Value.Const.proj axis modality in
         let is_constant = Mode.Modality.is_constant modality in
```

Comment:
this is a place where we're diverging (at least a little bit) from `kind-inference.md`. Can we either:

- update that to document that it's ok to treat non-constant modalities the same as identity modalities for the purposes of mode crossing in kinds
- add a comment here to the same effect?

Reply:
I've added a comment. Resolve if you're happy with it